Added documentation for the keba-charging-station component

dannerph opened the pull request:
**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24484

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9616"><img src="https://gitpod.io/api/apps/github/pbs/github.com/dannerph/home-assistant.io.git/53064ef882811ae09764b1ff9117894ef7fa7fe3.svg" /></a>

dannerph committed:
Added documentation for the keba-charging-station component

klaasnicolaas commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,97 @@
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.94
```
```suggestion
ha_release: 0.95
```

klaasnicolaas commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,97 @@
+  - Lock
+  - Sensor
+ha_release: 0.94
+redirect_from:
```
Please remove the `redirect_from`

dannerph replied:
Fixed, I probably used an old style documentation as template.

klaasnicolaas replied:
Well what I mainly see is that people unquestionably take over without really knowing what it is for 😉

dannerph committed:
Update source/_components/keba.markdown

push version number to 0.95

Co-Authored-By: Klaas Schoute <klaas_schoute@hotmail.com>

klaasnicolaas commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,97 @@
+      default: 0
+{% endconfiguration %}
+
+## {% linkable_title Services %}
```
Maybe put the service names in the titles

dannerph replied:
I am not 100% sure what your suggestion is, but I now added the service names in the descritpion of the service and additionally added some example payloads to the services calls with parameters.

dannerph committed:
added refresh_interval documentation and service call examples

dannerph committed:
bumped ha_release to 0.96, hopefully ;)

klaasnicolaas committed:
:pencil2: Tweak

- Remove linkable title (no longer needed)
- Put service name in the title/header

dannerph committed:
updated documentation for new services

dannerph committed:
small addition

MartinHjelmare commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,128 @@
+  - Binary Sensor
+  - Lock
+  - Sensor
+ha_release: 0.96
```
```suggestion
ha_release: 0.98
```

MartinHjelmare commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,128 @@
+
+### Authorizing and Deauthorizing `keba.authorize` and `keba.deauthorize`
+
+The charging station can be authorize and deauthorize vai service calls (`keba.authorize` and `keba.deauthorize`) or via the lock component that is created automatically for the charging station. In both cases the RFID tag from the configuration is used.
```
```suggestion
The charging station can be authorized and deauthorized via service calls (`keba.authorize` and `keba.deauthorize`) or via the lock component that is created automatically for the charging station. In both cases the RFID tag from the configuration is used.
```

MartinHjelmare commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,128 @@
+
+### Request New Data `keba.request_data`
+
+The service `keba.request_data` send data update requests to the charging station.
```
```suggestion
The service `keba.request_data` sends data update requests to the charging station.
```

MartinHjelmare commented on source/_components/keba.markdown:
```diff
@@ -0,0 +1,128 @@
+
+### Request New Data `keba.set_failsafe`
+
+The service `keba.set_failsafe` set the failsafe mode of the charging station. Payload example:
```
```suggestion
The service `keba.set_failsafe` sets the failsafe mode of the charging station. Payload example:
```

dannerph committed:
Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

dannerph committed:
Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

dannerph committed:
Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

dannerph committed:
Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

klaasnicolaas committed:
:pencil2: Tweak

klaasnicolaas committed:
:pencil2: Fix link

klaasnicolaas reviewed (dismissed):
Think we can merge it now

klaasnicolaas committed:
:pencil2: Last tweak
